pelicun/model/loss_model.py

```diff
@@ -1734,23 +1746,52 @@ def save_sample(
 
         return res
 
-    def load_sample(self, filepath: str | pd.DataFrame) -> None:
+    def load_sample(self, filepath: str | pd.DataFrame) -> dict[str, str]:
         """
         Load damage sample data.
 
+        Parameters
```

Comment:
commit: https://github.com/NHERI-SimCenter/pelicun/pull/51/commits/669a917420475eadf18ba7ac9de4a10858b8b007

I think there is a typo in the line above: this is loading loss sample data, not damage sample data.

Reply:
Correct. Thanks. a4d13561